prisma/migrations/01_functions/migration.sql

```diff
@@ -1,3 +1,4 @@
+SET GLOBAL log_bin_trust_function_creators = 1;
```

Comment:
Hmhm, still not sure about this one. This will cause migration to fail when we run it on a new environment if we don't do it as root, I running migrations as root has other problems.

Wouldn't it be better to run this specific line as a step in setting up a new environment (database server)?

Reply:
Oh, I'm so sorry, didn't intend committing this 😅 Nice spotted though!